### ISSUE TITLE
Running ./configure

### ISSUE THREAD

**Issue by svdhoog** [FLAME-HPC/libmboard#1]
The readme file states that the configuration can be run as follows:
`[root@stfc]# ./configure`
But on a freshly installed Ubuntu we had to run this to get it working:
`sudo sh ./configure`
Running with bash throws an error, because the configure file needs sh as shell:
`sudo bash ./configure`

**Comment by hosantosh** [comment]
Hi svdhoog,
I am also a novice user of libmboard. I need to install if for FLAME. I faced the same issue during libmboard configuration. The trick is to make the autogen.sh file executable using "chmod u+x autogen.sh" command. 

The error that I used to get was like this: 
/libmboard-master$ _sh ./autogen.sh
./autogen.sh: 12: ./autogen.sh: cannot create autogen.log: Permission denied
./autogen.sh: 14: ./autogen.sh: Syntax error: "(" unexpected_

However, after making it executable, it executes well only that there is another error message for README.in not being found!

![image](https://cloud.githubusercontent.com/assets/8624561/23860161/016d4e2c-07fe-11e7-9e32-c36cd858d5fc.png)

Now, I need to look for why is this happening! Any help from anybody.

Thanks in advance.

Sandeep

**Comment by svdhoog** [comment]
Can you tell us what is the output of: $ sudo sh ./configure

**Comment by hosantosh** [comment]
the configure file has not been created by autogen.sh.

**Comment by hosantosh** [comment]
![image](https://cloud.githubusercontent.com/assets/8624561/23860779/d351b51c-07ff-11e7-9a0d-b150223d02c8.png)

**Comment by hosantosh** [comment]
However, if I just create an empty "README.in" file by "touch README.in" everything works perfect. I get an "configure" file which runs fine. 

_$ sudo ./autogen.sh 
[sudo] password for sandeep: 
Running libtoolize ... DONE
Running aclocal    ... DONE
Running autoheader ... DONE
Running automake   ... DONE
Running autoconf   ... DONE
Overwriting 'COPYING' file ... DONE
sandeep@:~/Downloads/libmboard-master$_ 

![image](https://cloud.githubusercontent.com/assets/8624561/23861102/e84c35f4-0800-11e7-939a-1ded7a0660d3.png)

And, the trailing portion is 

![image](https://cloud.githubusercontent.com/assets/8624561/23861132/02cd49d6-0801-11e7-849b-86dc8c0d9108.png)

Looks good. But I have no idea whether everything is actually fine or not.

Best.

**Comment by hosantosh** [comment]
So, 
In summary, may be the authors/coders should change the README.txt or INSTALL.txt accordingly.
Because, after using these two additional commands/steps, the installation is very smooth, without errors. These commands are: 

_1) chmod u+x autogen.sh
2) touch README.in_

Finally, generate the configure file by simply running the 'autogen.sh' with 'sudo'.

**_1) sudo ./autogen.sh
2) ./configure
3) make
4) sudo make install_**

And, that is all.

**Comment by mondus** [comment]
Documents should be updated for this. @hosantosh Apologies for the delay we have had some issues with notifications.

**Comment by hosantosh** [comment]
No problem, mondus. 
But I found that installing FLAME is trickier than other softwares like NetLogo. 
So, I believe FLAME developers should also try to make the installation as simple as possible and without any glitches whatsoever.

**Comment by mondus** [comment]
@hosantosh Agreed. We have moved development to a completely open system now so if you have any improved install scripts you can submit these as pull requests. Otherwise we will use issues to track feature requests.

**Comment by hosantosh** [comment]
great.. 
and thanks.

**Comment by ptheywood** [comment]
As of 5fd34e8 following the updated instructions in the readme works correctly on Ubuntu 16.04.5 

https://github.com/FLAME-HPC/libmboard/blob/master/README.md#installation

```
./autogen.sh
./configure
make
sudo make install
```

**Comment by svdhoog** [comment]
Given this has been resolved, can the issue be closed?

**Comment by svdhoog** [comment]
In fact, I would propose to do a minor version release for libmboard 0.3.2 that includes this fix.
We are having similar problems with a FLAME model and using TravisCI, see here:
https://github.com/ETACE/eurace_unibi/pull/8
